README.md

```diff
@@ -12,21 +12,30 @@ Each window (i.e. split) gets its own personal history, and each time you create
 
 # Usage
 
-The default mappings are `<c-m>` to go back to the most previous buffer, and `<c-n>` to go to the next. Both of these mappings can be easily changed. Note that the `<c-m>` default will also override your carriage return mapping so you'll want to change that if you use `<cr>` for something else.
+This plugin gives `:Back` and `:Forward` commands to navigate similarly to a web browser.
 
 # Settings
 ```vim
 " Put these in your .vimrc (all are optional):
 
-" Use this setting to override the default mapping (<c-m>) for going back in the history
-nmap <C-M> <Plug>HistoryTraverseGoBack
-" Use this setting to override the default mapping (<c-n>) for going forward in the history
-nmap <C-N> <Plug>HistoryTraverseGoForward
+" Mappings
+nnoremap <leader>n :Back<CR>
+nnoremap <leader>m :Forward<CR>
+
 " Set filetypes to pass over putting in the history. Defaults to ['netrw']
 let g:history_ft_ignore = ['pyc', 'netrw']
+
 " Set the maximum length of each buffers history. Defaults to 100
 let g:history_max_len = 1000
+
+" Define characters used for indicator
+let g:history_indicator_back_active      = '⬅'
+let g:history_indicator_back_inactive    = '⇦'
+let g:history_indicator_forward_active   = '➡'
+let g:history_indicator_forward_inactive = '⇨'
+let g:history_indicator_separator        = ' '
 ```
+Note the indicator chars might look a little janky in your browser but should look okay in vim.
```

Comment:
I've found some icons that I really like for macvim, and I think once I merge this I'll make a change that uses `if has('gui_running')` to either pick my icons or yours. I think your choices do look okay in my terminal vim, but still not quite perfect - but I haven't found any that seem to work perfectly in the terminal.

Reply:
Hrmmm, mines perfect using Fira Code, but yeah this can be tricky. My active back arrow seems to be a little different then the others on some fonts. Not sure though.

<img width="133" alt="screen shot 2017-09-29 at 17 34 18" src="https://user-images.githubusercontent.com/8026227/31005545-8266c306-a53c-11e7-8ddb-f5a56266ef09.png">